homeassistant/components/aprilaire/entity.py

```diff
@@ -21,7 +21,7 @@ class BaseAprilaireEntity(BaseCoordinatorEntity[AprilaireCoordinator]):
     _attr_should_poll = False
 
     def __init__(
-        self, coordinator: AprilaireCoordinator, unique_id: str | None
+        self, coordinator: AprilaireCoordinator, unique_id: str | None = None
```

Comment:
Please don't add unneeded default values in the implementation to make it easier to add tests.

Reply:
Done